library/std/src/os/rtems/mod.rs

```diff
@@ -0,0 +1,4 @@
+#![stable(feature = "raw_ext", since = "1.1.0")]
+
+pub mod fs;
+pub mod raw;
```

Comment:
It has been our preference to not expose more instances of the `raw` module for new operating systems we add support for. Even if they claim to be a unix and thus must expose `std::os::unix::raw`, there should not be an accessible `std::os::rtems::raw`. These definitions do not magically become more correct or wise to implement and expose in std when we add new operating systems.

Reply:
@workingjubilee do you have any hints regarding this? I fail to find a solution which compiles.

Reply:
Can you just not make `raw` public?

Reply:
Thanks this worked.